src/app/blog/top-articles/top-article/top-article.component.html

```diff
@@ -9,9 +9,12 @@
     <span class="card-title activator grey-text text-darken-4">
       {{ article.title ? article.title : defaultText }}
     </span>
-    <p class="card-author-and-date">
+    <p *ngIf="languageService.currentLanguage === 'en'" class="card-author-and-date">
       {{ article.author ? article.author : defaultText }}・{{ (article.date ? article.date.toDate() : onInitDate) | date }}
     </p>
+    <p *ngIf="languageService.currentLanguage === 'es'" class="card-author-and-date">
```

Comment:
instead of having two p that optionally appear based on the language maybe we can have just one?
```html
<p class="card-author-and-date">
  {{ article.author ? article.author : defaultText }}・{{ (article.date ? article.date.toDate() : onInitDate) | date : 'mediumDate' : undefined : languageService.currentLanguage }}
</p>
```
not quite sure but the current language will be always es or es... eventually if we add another one this could work better i think.